packages/services/schema/src/orchestrators.ts

```diff
@@ -214,6 +277,16 @@ const createFederation: (
         } = await compose(subgraphs);
 
         if (composed.type === 'success') {
+          // merge all metadata from every subgraph by coordinate
+          let metadata: Record<string, Array<{ name: string; content: string }>> | null = null;
+          try {
+            const subgraphMetadatas = subgraphs.map(({ typeDefs }) => extractMetadata(typeDefs));
+            metadata = mergeMetadata(...subgraphMetadatas);
+          } catch (e: unknown) {
+            // warn and proceed because metadata isn't considered critical
+            logger.warn(`Cannot extract metadata from subgraphs because ${e}`);
+          }
```

Comment:
What could be thrown there?

Reply:
This is a remnant of a previous iteration. Ive removed the try/catch. It shouldn't throw.